examples/default.yaml

```diff
@@ -61,11 +61,28 @@ mounts:
     # system will look and feel like regular files directories in the Guest OS.
     # 🟢 Builtin default: false
     followSymlinks: null
+  virtfs:
+    # Supported security models are "passthrough", "mapped-xattr", "mapped-file" and "none".
+    # 🟢 Builtin default: "mapped-xattr"
+    securityModel: null
+    # Select 9P protocol version. Valid options are: "9p2000" (legacy), "9p2000.u", "9p2000.L".
+    # 🟢 Builtin default: "9p2000.L"
+    protocolVersion: null
+    # The number of bytes to use for 9p packet payload, 4 KiB is minimum and 8 KiB is default.
+    # 🟢 Builtin default: "128 KiB"
+    msize: null
+    # Specifies a caching policy. Valid options are: "none", "loose", "fscache" and "mmap".
+    # 🟢 Builtin default: "none"
+    cache: null
 - location: "/tmp/lima"
   # 🟢 Builtin default: false
   # 🔵 This file: true (only for "/tmp/lima")
   writable: true
 
+# Mount type for above mounts, such as "reverse-sshfs" (from sshocker) or "9p" (EXPERIMENTAL, from QEMU’s virtio-9p-pci, aka virtfs)
+# 🟢 Builtin default: "reverse-sshfs"
+mountType: null
```

Comment:
Is there a reason this has to be global? I guess having a global default makes sense, but could it be overridable on a per-mount basis?

Reply:
I've wondered this for a while now myself but haven't figured out where to discuss it.

Reply:
What would the use case be for using two different mount types simultaneously? I assume that a user would determine which works best for them given their existing installed dependencies and would generally stick to that method, and at the moment `sshfs` is the tried-and-true method, users can try out the other method instead, and they should generally be drop-in compatible, right?

Reply:
Right now the main use case would be side-by-side comparisons from within the same VM without having to reboot to switch back and forth.

I hope that eventually there will be no reason to keep sshfs at all, so this becomes moot.

But if in the interim there are reasons why sometimes sshfs is still useful/better than 9p, why not allow the user using both methods at the same time (as long as the effort to enable this is pretty minimal).

So this is not a strong request, but more a convenience for the transition period.

Reply:
Yes, for YAML definition we can make it non-global.

But for ease of implementation we can error out when multiple mount types are mixed in, and call it a day.

Reply:
The plan was that the default could change to 9p, but that the older distributions could use sshfs.

Similar to e.g. `legacyBIOS`.